examples/sensor-data-java/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.2.8
+sbt.version = 1.3.0
```

Comment:
We are keeping SBT at `1.2.8` in all projects for now.

Reply:
https://github.com/sbt/sbt-java-formatter/commit/48bb4c45d7b71a128652315d8f15b6ec9117510c @olofwalker

Reply:
Seems like it is time to upgrade to 1.3.x. We will pick that up ASAP to unblock this PR